sdk/script/management-sdk.ts

```diff
@@ -169,6 +169,11 @@ class AccountManager {
         return this.post(urlEncode `/apps/${appName}/deployments/`, JSON.stringify(deployment), /*expectResponseBody=*/ true)
             .then((res: JsonResponse) => res.body.deployment);
     }
+    
+    public clearDeployment(appName: string, deploymentName: string): Promise<void> {
```

Comment:
Aren't those two names basically semantically equivalent? The deployment is essentially just a history of releases, so I believe simply calling it clear is clear enough. I don't feel super strongly about the SDK name, but I definitely am in favor of keeping the CLI command "code-push deployment clear" as opposed to "code-push deployment clear-history" or something like that.